starter-code/css/main.css

```diff
@@ -6,5 +6,61 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  border-top: 250px solid #125688;
+  margin: 20px 20px 20px 20px;
```

Comment:
Try not to apply styling like margins within the entire body element - instead, apply it to individual elements that require it. In this case, the margin 'boxes' all of your elements into the center of the page surrounded by white padding.